supervisor/addons/validate.py

```diff
@@ -224,6 +227,23 @@ def _migrate(config: Dict[str, Any]):
                 )
             config[ATTR_TMPFS] = True
 
+        # 2021-06 "snapshot" renamed to "backup"
+        for entry in (
+            ATTR_SNAPSHOT_EXCLUDE,
+            ATTR_SNAPSHOT_POST,
+            ATTR_SNAPSHOT_PRE,
+            ATTR_SNAPSHOT,
```

Comment:
```suggestion
            "snapshot_exclude",
            "snapshot_post",
            "snapshot_pre",
            "snapshot",
```

No need to keep it as an attribute if we want to ditch it. Because this code stay maybe for the next 5 years around